client_internals_it_test.go

```diff
@@ -421,6 +423,32 @@ func clientInternalEncodeDataTest(t *testing.T) {
 	})
 }
 
+func clientClusterConnectionConfigRetryTimeTest(t *testing.T) {
+	// TODO: Adapt this test for t.Parallel()
+	//t.Parallel()
+	ctx := context.Background()
+	const ASSERTION_SECONDS = 30
+	port := it.NextPort()
+	cls := it.StartNewClusterWithOptions("dev", port, 1)
+	defer cls.Shutdown()
+	config := cls.DefaultConfig()
+	config.Cluster.ConnectionStrategy.Retry.InitialBackoff = math.MaxInt32
+	config.Cluster.ConnectionStrategy.Retry.MaxBackoff = math.MaxInt32
+	client := it.MustClient(hz.StartNewClientWithConfig(ctx, config))
```

Comment:
Could you defer shutdown the client?

Reply:
Added at https://github.com/hazelcast/hazelcast-go-client/pull/861/commits/c167514ad9aaae7d2c8ed4be7311bbc1962ab04b.

Reply:
Shutdown client without waiting added at https://github.com/hazelcast/hazelcast-go-client/pull/861/commits/28ac57dcbfc8759496594fe4f12942fdfc4a4a0e.